install/kubernetes/README.md

```diff
@@ -133,34 +133,36 @@ All configurable variables and default values can be seen in `values.yaml`, with
 Typically, you may want to configure the following blocks and options:
 
 * Global part is mandatory and contain attributes like `appName` of your install,
-* `microcks` part is mandatory and contain attributes like the number of `replicas` and the access `url` if you want some customizations, 
+* `microcks` part is mandatory and contain attributes like the number of `replicas` and the access `url` if you want some customizations,
 * `postman` part is mandatory for the number of `replicas`
 * `keycloak` part is optional and allows specifying if you want a new install or reuse an existing instance,
 * `mongodb` part is optional and allows specifying if you want a new install or reuse an existing instance.
 * `features` part is optional and allows enabling and configuring opt-in features of Microcks.
 
-The table below describe all the fields of the `values.yaml`, providing informations on what's mandatory and what's optional as well as default values.
+The table below describes all the fields of the `values.yaml`, providing information on what's mandatory and what's optional as well as default values.
 
 | Section       | Property           | Description   |
 | ------------- | ------------------ | ------------- |
-| `microcks`    | `url`              | **Mandatory**. The URL to use for exposing `Ingress` | 
+| `microcks`    | `url`              | **Mandatory**. The URL to use for exposing `Ingress` |
 | `microcks`    | `ingressSecretRef` | **Optional**. The name of a TLS Secret for securing `Ingress`. If missing, self-signed certificate is generated. |
-| `microcks`    | `ingressAnnotations`  | **Optional**. A map of annotations that will be added to the `Ingress` for Microcks main pod. If these annotations are triggering a Certificate generation (for example through https://cert-manager.io/). The `generateCert` property should be set to `false`. | 
+| `microcks`    | `ingressAnnotations`  | **Optional**. A map of annotations that will be added to the `Ingress` for Microcks main pod. If these annotations are triggering a Certificate generation (for example through [cert-mamanger.io](https://cert-manager.io/)). The `generateCert` property should be set to `false`. |
 | `microcks`    | `generateCert`     | **Optional**. Whether to generate self-signed certificate or not if no valid `ingressSecretRef` provided. Default is `true` |
 | `microcks`    | `replicas`         | **Optional**. The number of replicas for the Microcks main pod. Default is `1`. |
 | `microcks`    | `image`            | **Optional**. The reference of container image used. Chart comes with its default version. |
+| `microcks`    | `service`          | **Optional**. Some service spec values to use. Supports `type` which defaults to `ClusterIP`. |
```

Comment:
Seems like you've used `.Values.microcks.serviceType` in Yaml templates but are mentioning here just `service` with `type` property. Could you please re-align both parts? `.Values.microcks.service.type` sounds nice to me. 

BTW: thanks for all the typo fixes ;-)

Reply:
Sorry, my mistake. I forgot to update the Readme on my last push.
Using `.Values.microcks.service.type` causes issues. The way the templates are evaluated means it fails if there is no service section in the Values file: `Error: template: microcks/templates/service.yaml:15:18: executing "microcks/templates/service.yaml" at <.Values.microcks.service.type>: nil pointer evaluating interface {}.type`
I think we have two options. Option 1, I update the Readme to reflect `.Values.microcks.serviceType`.
Option 2, we make service mandatory in values.yaml, but leave it empty by default, something like this:
```
  service: {
    # Uncomment to change Keycloak Service to a NodePort
    #type: NodePort
  }
```
I'll update the PR with option 1 as I think it is less messy. It might be better to move to option 2 if there are calls to override more of the service spec in the future. Let me know if you prefer option 2 for this PR though.

Reply:
Updated: 73c86a0